src/behavior/src/A32NX_Interior_ISIS.xml

```diff
@@ -52,11 +52,15 @@
                         0 (&gt;H:A32NX_ISIS_KNOB_ANTI_CLOCKWISE)
                     } els{
                         (L:A32NX_ISIS_BARO_MODE, enum) #MODE_STD# != if{
+<<<<<<< HEAD
```

Comment:
You have some issues from resolving conflicts I see.

Reply:
I didn't mean to touch this file...must have missed it while resolving conflicts when merging to master.